bot/sinusbot/Dockerfile

```diff
@@ -1,16 +1,20 @@
 FROM        --platform=$TARGETOS/$TARGETARCH debian:stable-slim
 
-LABEL maintainer="ki2007 <ki2007@damw.eu>" version="1.0"
+LABEL       org.opencontainers.image.authors="i2007@damw.eu"
+LABEL       version="1.0"
 
 ENV         DEBIAN_FRONTEND noninteractive
 
 # Install Dependencies
 RUN         apt update \
             && apt upgrade -y \
-            && apt install -y ca-certificates less locales pulseaudio python python3 sudo x11vnc x11-xkb-utils xvfb iproute2 \
+            && apt install -y ca-certificates less locales pulseaudio python python3 sudo x11vnc x11-xkb-utils xvfb iproute2 ffmpeg curl \
```

Comment:
Can you also add `bzip2 libxss1 libxcomposite1` as there mentioned in the docs that the are needed:
https://sinusbot.github.io/docs/installation/linux/

Reply:
done